src/utils/utils_cache_line.c

```diff
@@ -167,3 +167,40 @@ void set_cache_line_dirty(struct ocf_cache *cache, uint8_t start_bit,
 
 	ocf_cleaning_set_hot_cache_line(cache, line);
 }
+
+void cline_rebuild_metadata(ocf_cache_t cache,
```

Comment:
I think our convention is to always have externally linked function names prefixed with "ocf_"